scss/components/alert.scss

```diff
@@ -1,183 +1,225 @@
+      @include fd-var-color("color", $fd-alert-border-color--warning, --fd-alert-border-color);
     }
-    &--error {
-      --fd-alert-background-color: var(--fd-color-background-negative);
-      --fd-alert-border-color: var(--fd-color-status-3);
-      &::before,
-      &::after {
-        @include fd-var-color("color", $fd-alert-border-color--error, --fd-alert-border-color);
-      }
-      @include fd-var-color("background-color", $fd-alert-background-color--error, --fd-alert-background-color);
-      @include fd-var-color("border-color", $fd-alert-border-color--error, --fd-alert-border-color);
-      @include fd-icon("message-error", "m");
-      @include fd-rtl() {
-        @include fd-icon("message-error", "m", "after");
-      }
+
+    @include fd-var-color("background-color", $fd-alert-background-color--warning, --fd-alert-background-color);
+    @include fd-var-color("border-color", $fd-alert-border-color--warning, --fd-alert-border-color);
+    @include fd-icon("message-warning", "m");
+
+    @include fd-rtl() {
+      @include fd-icon("message-warning", "m", "after");
     }
-    &--success {
-      --fd-alert-background-color: var(--fd-color-background-positive);
-      --fd-alert-border-color: var(--fd-color-status-1);
-      &::before,
-      &::after {
-        @include fd-var-color("color", $fd-alert-border-color--success, --fd-alert-border-color);
-      }
-      @include fd-var-color("background-color", $fd-alert-background-color--success, --fd-alert-background-color);
-      @include fd-var-color("border-color", $fd-alert-border-color--success, --fd-alert-border-color);
-      @include fd-icon("message-success", "m");
-      @include fd-rtl() {
-        @include fd-icon("message-success", "m", "after");
-      }
+  }
+
+  &--error {
+    --fd-alert-background-color: var(--fd-color-background-negative);
+    --fd-alert-border-color: var(--fd-color-status-3);
+
+    &::before,
+    &::after {
+      @include fd-var-color("color", $fd-alert-border-color--error, --fd-alert-border-color);
     }
-    &--information {
-      --fd-alert-background-color: var(--fd-color-background-information);
-      --fd-alert-border-color: var(--fd-color-action-1);
-      &::before,
-      &::after {
-        @include fd-var-color("color", $fd-alert-border-color--information, --fd-alert-border-color);
-      }
-      @include fd-var-color("background-color", $fd-alert-background-color--information, --fd-alert-background-color);
-      @include fd-var-color("border-color", $fd-alert-border-color--information, --fd-alert-border-color);
-      @include fd-icon("message-information", "m");
-      @include fd-rtl() {
-        @include fd-icon("message-information", "m", "after");
-      }
+
+    @include fd-var-color("background-color", $fd-alert-background-color--error, --fd-alert-background-color);
+    @include fd-var-color("border-color", $fd-alert-border-color--error, --fd-alert-border-color);
+    @include fd-icon("message-error", "m");
+
```

Comment:
why is this space created?

Reply:
I'm pretty sure it's because the next declaration has an open block `{ }`. I'm guessing it's a different _style_ of include, so it helps to visually separate it.
The equivalent in JS would be wanting a line break between these two:

```
const thing = 1;

function () {
  console.log(thing);
}
```